geoext.ux/ux/SimplePrint/lib/GeoExt.ux/SimplePrint.js

```diff
@@ -429,10 +430,12 @@ GeoExt.ux.SimplePrint = Ext.extend(Ext.form.FormPanel, {
      *  Do the print.
      */
     print: function() {
+        // TODO cancel ongoing print job if any
```

Comment:
If the user hits "Print" again, before the previous job has finished, you want to cancel the previous job.